app/models.py

```diff
@@ -12,110 +28,44 @@ def strip_services_wrapper(content):
-    res = es.search(index="services",
-                    body={
-                        "from": start_from, "size": size,
-                        "fields": ["id", "serviceName", "lot",
-                                   "serviceSummary"],
-                        "query": {
-                            "filtered": {
-                                "query": {
-                                    "multi_match": {
-                                        "query": query,
-                                        "fields": ["serviceName",
-                                                   "serviceSummary",
-                                                   "serviceFeatures",
-                                                   "serviceBenefits"],
-                                        "operator": "or"
-                                    }
-                                },
-                                "filter": {
-                                    "term": {"lot": lot.lower()}
-                                }
-                            }
-                        }
-                    }
-                    )
-    return res
+    url = search_url + "/search?q=" + query
+    response = requests.get(
+        url,
+        headers={
+            "authorization": "Bearer {}".format(search_access_token)
+        }
+    )
+    return response.content
 
 
-def search_with_filters(query, args, start_from=0, size=10):
-    filters = []
-    for k, v in args.iteritems():
-        filters.append({"term": {k: v.lower()}})
-    res = es.search(index="services",
-                    body={
-                        "from": start_from, "size": size,
-                        "fields": ["id", "serviceName", "lot",
-                                   "serviceSummary"],
-                        "query": {
-                            "filtered": {
-                                "query": {
-                                    "multi_match": {
-                                        "query": query,
-                                        "fields": ["serviceName",
-                                                   "serviceSummary",
-                                                   "serviceFeatures",
-                                                   "serviceBenefits"],
-                                        "operator": "or"
-                                    }
-                                },
-                                "filter": {
-                                    "bool": {"must": filters}
-                                }
-                            }
-                        }
-                    }
-                    )
-    return res
+def search_with_filters(query, filters, start_from=0, size=10):
+    filter_string = ""
+    for k, v in filters.iteritems():
+        filter_string = filter_string + "&" + k + "=" + v.lower()
+    url = search_url + "/search?q=" + query + filter_string
+    response = requests.get(
```

Comment:
Query parameters can be passed directly to requests. This is safer than joining them ourselves because things like query escaping is done for us. (same point for line 43 as well)